jquery.viewport.js

```diff
@@ -114,7 +114,7 @@
         return true;
     }
 
-    $.extend($.expr[':'], {
+    $.extend($.expr.pseudos, {
```

Comment:
This breaks jQuery 1.5.x, which is fine because the package doesn't say which version of jQuery it supports, but as an easy way to ignore it, would you mind changing this to `$.expr.pseudos || $.expr[':']`?